hack/bazel-build-functests.sh

```diff
@@ -23,6 +23,13 @@ source hack/common.sh
 source hack/bootstrap.sh
 source hack/config.sh
 
+# Remove the suppress-focus-container ginkgolinter flag when running e2e tests
+# to avoid focus containers sneaking past and being accidentally committed
+if [ -n "${KUBEVIRT_E2E_FOCUS}" ]; then
```

Comment:
If the intention is to act based on CI, isn't `CI` the correct env var to check against?
This `*FOCUS` one has a total different purpose (filtering which tests to run).

There may be another problem with this change, but I am not sure if the effect is visible or just theoretical: The modified file is part of BAZEL config, i.e. one of its input, therefore, touching it may invalidate its cache and take out one of BAZEL main advantages. That cache is shared between running instances on CI.

IMO this is a nasty hack, I do not think it is worth it. Hacks usually bite back sooner or later.
There is an elegant solution to this, to run an out-of-bazel check as suggested originally.

Reply:
@lyarwood @EdDev Exactly, we have `"$CI"` which is set to `true` when running in CI env

Reply:
ACK forgot to set this in the latest run but I'll change it once the compute test has finished.